l1-contracts/src/core/Rollup.sol

```diff
@@ -381,6 +385,69 @@ contract Rollup is Leonidas, IRollup, ITestRollup {
     return blocks[_blockNumber].archive;
   }
 
+  /**
+   * @notice  Check if a proposer can propose at a given time
+   *
+   * @param _ts - The timestamp to check
+   * @param _proposer - The proposer to check
+   * @param _archive - The archive to check (should be the latest archive)
+   *
+   * @return uint256 - The slot at the given timestamp
+   * @return uint256 - The block number at the given timestamp
```

Comment:
The wording here probably got slightly weird. But it is the block number at the timestamp, e.g., the next block. I am inclined to fix it in a later pr along with other things because of ci times.